docs/docs/themes/shadowing.md

```diff
@@ -15,28 +15,27 @@ If you’ve installed `gatsby-theme-blog` you’ll notice that it renders a `Bio
 You can inspect `gatsby-theme-blog`'s file structure to determine the file path for the file you want to shadow.
 
 ```text:title="tree gatsby-theme-blog"
```

Comment:
I know this predated this PR, but the title here only renders the word "tree" and not the "gatsby-theme-blog" part.